### ISSUE TITLE
Portal Web UI doesn't refresh Function code

### ISSUE THREAD

**Issue by ghost** [Azure/Azure-Functions#759]
When I redeploy a function to Azure, it's updated and works well. but in Azure Portal it still shows the old version of Function code. Maybe some caching issue. Could you please check this?

**Comment by davidebbo** [comment]
Yes, it's a known issue. You just need to click the little refresh button next to the Function App name.

**Comment by iredjee** [comment]
It shows infinite loading when I press refresh button near app service name.

![screenshot from 2018-04-20 19-04-50](https://user-images.githubusercontent.com/5600315/39058828-f54e291e-44cd-11e8-8640-3588e93c14ff.png)

**Comment by hxlnt** [comment]
+1 on these errors. The refresh icon, Save/Save and Run buttons seem to have no effect after editing a function in the portal. Restarting entire web app works sometimes but not always. In my case, this is with a function generated from the portal and not connected to a deployment source.

**Comment by ColbyTresness** [comment]
@mattchenderson for portal

**Comment by softcraftsman** [comment]
@ColbyTresness @mattchenderson  - Do we have any updates or ETA on this bug fix?  This is causing problems with our users.
